doc/manual/packages.rst

```diff
@@ -814,18 +806,19 @@ It also creates a new version entry in your local ``METADATA`` repo for ``FooBar
     @@ -0,0 +1 @@
     +84b8e266dae6de30ab9703150b3bf771ec7b6285
 
-If there is a ``REQUIRE`` file in your package repo, it will be copied into the appropriate spot in ``METADATA`` when you tag a version.
-Package developers should make sure that the ``REQUIRE`` file in their package correctly reflects the requirements of their package, which will automatically flow into the official metadata if you're using :func:`Pkg.tag`.
-See the `Requirements Specification <#man-package-requirements>`_ for the full format of ``REQUIRE``.
+The :func:`Pkg.tag` command takes an optional second argument that is either an explicit version number object
+like ``v"0.0.1"`` or one of the symbols ``:patch``, ``:minor`` or ``:major``.  These increment the patch, minor
+or major version number of your package intelligently.
+
+Adding a tagged version of your package will expedite the official registration into METADATA.jl by collaborators. It is
+strongly recommended that you complete this process, regardless if your package is completely ready for an official release.
```

Comment:
Actually nevermind maybe not, since the "Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable." bullet is easy to miss. Hopefully people can figure out when is best to do minor vs patch.